packages/health-check/src/schedule.js

```diff
@@ -1,30 +1,45 @@
-const schedule = require("node-schedule");
+const { CronJob } = require("cron");
+const ms = require("ms");
```

Comment:
I was gonna say that adding a new package just to avoid writing `60 * 1000` is a bit too much but it's a 6KB package with some nice functionality. Nice one.

Reply:
:)